wayback/_client.py

```diff
@@ -112,6 +112,61 @@ def __init__(self, retries, total_time, causal_error):
     'raw_url',
     'view_url'
 ))
+"""
+Item from iterable of results returned by :meth:`WaybackClient.search`
+
+These attributes contain information provided directly by CDX.
+
+.. py:attribute:: digest
+
+   Hash of captured content given as a string, such as
+   :data:`'MGIGF4GRGGF5GKV6VNCBAXOE3OR5BTZC'`.
+
+.. py:attribute:: key
+
+   TODO: Something related to SURT?
+
+.. py:attribute:: length
+
+   Size of captured content in bytes, given as a string such as :data:`'2767'`.
+
+.. py:attribute:: mime_type
+
+   MIME type of record, such as :data:`'text/html'`.
```

Comment:
May be `'warc/revisit'` if a revisit record, even though useful mime type info is contained elsewhere. May also be `'unk'` if this data wasn’t captured for some reason.